sdk/python/foundation-models/system/finetune/question-answering/extractive-qa.ipynb

```diff
@@ -40,6 +40,42 @@
     "* Check or create compute. A single GPU node can have multiple GPU cards. For example, in one node of `Standard_ND40rs_v2` there are 8 NVIDIA V100 GPUs while in `Standard_NC12s_v3`, there are 2 NVIDIA V100 GPUs. Refer to the [docs](https://learn.microsoft.com/en-us/azure/virtual-machines/sizes-gpu) for this information. The number of GPU cards per node is set in the param `gpus_per_node` below. Setting this value correctly will ensure utilization of all GPUs in the node. The recommended GPU compute SKUs can be found [here](https://learn.microsoft.com/en-us/azure/virtual-machines/ncv3-series) and [here](https://learn.microsoft.com/en-us/azure/virtual-machines/ndv2-series)."
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Install dependencies by running below cell. This is not an optional step if running in a new environment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install azure-ai-ml==1.5.0\n",
+    "%pip install azure-identity\n",
+    "%pip install datasets==2.9.0"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To find number of GPU's in a SKU run the below cell by replacing the `<values>` with appropriate values. You can change the compute name based on the SKU you selected. Below is an example for `Standard_ND40rs_v2` SKU."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!az ml compute list-sizes -g \"<RESOURCE_GROUP>\" -w \"<WORKSPACE_NAME>\" --subscription \"<SUBSCRIPTION_ID>\" --query \"[?name=='Standard_ND40rs_v2'].{Name:name, Gpus:gpus}\" --output table"
```

Comment:
don't run cli in the notebook if not necessary. there is a python for this. https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.operations.computeoperations?view=azure-python#azure-ai-ml-operations-computeoperations-list-sizes.. 

ideally i want you to query the compute cluster created in earlier step and get the sku. From the sku, query the number of gpu using list size and set the gpu count on user behalf so user does not have to set it manually.

Reply:
will add it